capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml

```diff
@@ -131,4 +132,13 @@
         </LinearLayout>
 
     </LinearLayout>
+
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/compose_view"
```

Comment:
[Request] Please apply changes to tablet layouts too.

Reply:
This component was deleted